src/addRemove.js

```diff
@@ -0,0 +1,121 @@
+export const appendTask = (text, size) => {
+  const child = document.createElement('li');
+  child.setAttribute('class', 'item itemList');
+  child.setAttribute('id', size);
+  child.innerHTML = `
+  <input class='check' type='checkbox'>
+  <label class=" visible itemText"> ${text} </label>
+  <input class="editText" type="text">
+  <button class=" visible btn-dots">
+      <span class="dots">&#8942;</span>
+  </button> 
+  <button class="btn-save" >Save</button>
+  <button class="btn-close">
+      <span class="close">&#215;</span>
+  </button>`;
+  document.querySelector('.list').appendChild(child);
+};
+
+export const addByOrder = (text) => {
+  let size = 0;
+  if (localStorage.getItem('list') != null) {
+    const arr = JSON.parse(localStorage.getItem('list'));
+    arr.push(text);
+    localStorage.setItem('list', JSON.stringify(arr));
+    size = arr.length - 1;
+  } else {
+    const newArr = [];
+    newArr.push(text);
+    localStorage.setItem('list', JSON.stringify(newArr));
+  }
+  appendTask(text, size);
+};
```

Comment:
- [ ] When adding a task to the array, each task should be added as an object containing the following key pair values as shown in the example code below. Each task should contain a `description, completed & index`. Currently for you project you are only storing the text as a value not as an object. I kindly suggest you see the project requirement below and make the necessary changes 
```javascript
 {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
```
> - Project requirement
![re](https://github.com/BiancaMe/ToDoProject/assets/89055502/67b642ee-7509-4cfe-8070-12c82b630562)
- [ ] Ensure that when adding & deleting a task the index is reordered correctly as required by the above project requirement 👍

Reply:
This change has been done well